pages/index.js

```diff
@@ -1,7 +1,49 @@
+console.clear();
+
+import { useState } from "react";
+import styled from "styled-components";
+
+import { ideas } from "@/lib/db";
+import Card from "../components/Card";
+import Header from "@/components/Header";
+
+const defaultIdeas = ideas;
+
+const CardContainer = styled.ul`
```

Comment:
```suggestion
const CardList = styled.ul`
```
If you're working with ListItem as a suffix, then I'd rename this to CardList.